Packs/Slack/Integrations/SlackV3/SlackV3.py

```diff
@@ -1063,20 +1004,44 @@ async def handle_listen_error(error: str):
     demisto.updateModuleHealth(error)
 
 
-async def start_listening():
+def start_listening():
     """
     Starts a Slack SocketMode client and checks for mirrored incidents.
     """
     try:
-        executor = concurrent.futures.ThreadPoolExecutor(max_workers=1)
-        loop = asyncio.get_running_loop()
-        loop.run_in_executor(executor, long_running_loop)
-        await slack_loop()
+        # create the long-running background task loop
+        background_task = threading.Thread(target=long_running_loop, daemon=True)
+        background_task.start()
+        demisto.info("Created background task")
+
+        # Create the message listeners
+        message_consumers = [
+            threading.Thread(target=consume_and_process_message, args=(MESSAGE_QUEUE, i, CLIENT), daemon=True) for i
+            in range(3)]
```

Comment:
The `3` should be a parameter. Preferably something like an advanced/hidden parameter.